package.json

```diff
@@ -9,11 +9,22 @@
     "storybook": "cd ui && start-storybook -p 6006",
     "postinstall": "cd ui && npm i && cd ../api && npm i",
     "lint": "prettier --ignore-path .gitignore --check . && eslint .",
-    "fix": "prettier --ignore-path .gitignore --write . && eslint . --fix"
+    "fix": "prettier --ignore-path .gitignore --write . && eslint . --fix",
+    "test:run:mongo": "sudo docker rm -fv mongo-dreams-test ; sudo docker run --name mongo-dreams-test --publish=28017:27017 mongo:4.4.3",
```

Comment:
We should remove this line obviously but from when I wrote this for realities I remember that the whole `rm` thing was important to not end up with tons of leftover docker containers or something, it eats up your storage space otherwise unless you manually run commands to clean up stuff. Maybe not a big problem if we'll mostly run the tests in ci though

Reply:
So wondering if there's a simple equivalent for docker-compose

Reply:
Maybe this from a few lines above is a good hint 

```
"db:up": "docker-compose up",
"db:reset": "docker-compose down && docker-compose rm -f",
```

Reply:
I think for now it"s fine to not have a command to clean it since we're using the same db instance for normal development. We'll handle this in the future when we create a new db to be able to wipe it afterwards